runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -33,7 +33,7 @@
             "pip": [
                 f"prophet=={prophet.__version__}",
                 f"cloudpickle=={cloudpickle.__version__}",
-                f"databricks-automl-runtime==0.2.5",
```

Comment:
+1

Reply:
The issue to use the version from version.py is that in the env file of the model will include a package that not on pypi. It may cause some issue for our testing with the main branch of the repo. But it is fine on the released ones.